rbc/request_internal_test.go

```diff
@@ -0,0 +1,49 @@
+package rbc
+import (
+	"reflect"
+	"testing"
+)
+
+func TestValReqRepository_Find(t *testing.T) {
+	valrepo,_:= NewValReqRepository()
+	valstruct:=ValRequest{nil,nil,nil}
+	valrepo.Save("ID1",&valstruct)
+	if valrepo.Find("ID1")!=&valstruct{
+		t.Fatalf("request %v is not found.",valrepo.recv["ID1"])
+	}
+	if len(valrepo.FindAll())!=1{
+		t.Fatalf("request %v is not found.",valrepo.recv["ID1"])
+	}
+
+}
+func TestValReqRepository_FindAll(t *testing.T) {
+	valrepo,_:=NewValReqRepository()
+	valstruct:=ValRequest{nil,nil,nil}
+	valrepo.Save("ID1",&valstruct)
+	valrepo.Save("ID2",&valstruct)
+	if len(valrepo.FindAll())!=2{
+		t.Fatalf("request %v,%v is not found all.",valrepo.recv["ID1"],valrepo.recv["ID2"])
+	}
+	for connId,_:= range valrepo.recv{
```

Comment:
Because this function is FindAll test, I think you need to check whether correct value has returned from FindAll function

Reply:
I checked the correct value too. I checked the length and value both.

Reply:
Yes, but it's valrepo's value, Not a value from FindAll() func.

Reply:
OK.